src/document.h

```diff
@@ -159,7 +170,9 @@ struct Document {
 
     const wxChar *SaveDB(bool *success, bool istempfile = false, int page = -1) {
         if (filename.empty()) return _(L"Save cancelled.");
-
+        uint xs = selected.xs;

```

Comment:
if variables are used once, just remove them, and use their values below

Reply:
Ok, I corrected this. Thanks!